pkg/scannerv4/mappers/mappers.go

```diff
@@ -695,6 +762,47 @@ func pkgFixedBy(enrichments map[string][]json.RawMessage) (map[string]string, er
 	return pkgFixedBys, nil
 }
 
+// cveEPSS unmarshals and returns the EPSS enrichment, if it exists.
+func cveEPSS(ctx context.Context, enrichments map[string][]json.RawMessage) (map[string]epss.EPSSItem, error) {
+	enrichmentList := enrichments[epss.Type]
+	if len(enrichmentList) == 0 {
+		zlog.Warn(ctx).
+			Str("enrichments", epss.Type).
+			Msg("no EPSS enrichments found")
+		return nil, nil
+	}
+
+	var epssMap map[string][]epss.EPSSItem
+	err := json.Unmarshal(enrichmentList[0], &epssMap)
+	if err != nil {
+		return nil, fmt.Errorf("error unmarshaling EPSS enrichment: %w", err)
+	}
+
+	if len(epssMap) == 0 {
+		return nil, nil
+	}
+
+	res := make(map[string]epss.EPSSItem)
+	for p, details := range epssMap {
```

Comment:
what is `p`? The enricher maps vulnerability IDs to EPSS scores. Perhaps replace `p` with `vulnID` or `id` or something like that